rules-topics/999_changelog.md

```diff
@@ -23,14 +23,17 @@
 ### Effects
 * BLEED removed.
 * LIFE updated (no longer provides immunity).
+* UNCONSCIOUS now lasts 1 minute instead of 5.
```

Comment:
Is this reflected in life and death doc?

Reply:
yes